robot/EDA/resources/CourseConnectionsSettingsPageObject.py

```diff
@@ -20,3 +21,50 @@ def _is_current_page(self):
             locator_tab,
             error=f"Course Connections tab with locator '{locator_tab}' is not available on the page"
         )
+
+    def set_enable_course_connections(self):
```

Comment:
what will happen if checkbox is already checked? may be first check if its checked already and if its not checked and check it

Reply:
That's why as part of the test setup, we are making sure the checkbox is defaulted to false. 

The reason why I am deferring this is we can't check the status of the checkbox when we are in edit mode. To catch this, I will have to go out of edit mode, check the status, click edit and toggle accordingly. Going out of edit mode is complicated, coz do I Save or Cancel ? If I Save, will it cause issues because I might not have completed all mandatory fields in my .robot file etc.

So a solution to this is: I have requested a change in the DOM and it is currently under SPEC review. I anticipate it to be available to us in the next week or so. And then I will update the code to check the status and THEN toggle. Sound okay to you ?